modules/provision-oci-infrastructure-ocp-cluster.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-assisted-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="provision-oci-infrastructure-ocp-cluster_{context}"]
+= Provisioning {oci} infrastructure for your cluster
+
+By using the {ai-full} to create details for your {product-title} cluster, you can specify these details in a stack. A stack is an {oci} feature where you can automate the provisioning of all necessary {oci} infrastructure resources, such as the custom image, that are required for installing an {product-title} cluster on {oci}.
+
+The {oci-first} Compute Service creates a virtual machine (VM) instance on {oci}. This instance can then automatically attach to a virtual network interface controller (vNIC) in the virtual cloud network (VNC) subnet. On specifying the IP address of your {product-title} cluster in the custom manifest template files, the {oci} instance can communicate with your cluster over the VNC.
+
+.Prerequisites
+
+* You uploaded the discovery ISO image to the {oci} bucket. For more information, see "Using the {ai-full} to generate an {oci}-compatible discovery ISO image".
+
+.Procedure
+
+. Complete the steps for provisioning {oci} infrastructure for your {product-title} cluster. See link:https://docs.oracle.com/iaas/Content/openshift-on-oci/installing-assisted.htm#install-cluster-apply-stack["Creating OpenShift Container Platform Infrastructure Using Resource Manager" (Oracle documentation)].
```

Comment:
```suggestion
. Complete the steps for provisioning {oci} infrastructure for your {product-title} cluster. See link:https://docs.oracle.com/iaas/Content/openshift-on-oci/installing-assisted.htm#install-cluster-apply-stack["Creating OpenShift Container Platform Infrastructure Using Resource Manager"] in the Oracle documentation.
```

Reply:
OK as per recent guideline updates https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-xref-attribute